distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -497,6 +508,7 @@ type Table
         self.updated_context new_ctx
 
     ## UNSTABLE
+       ALIAS Add Column, Update Column
 
```

Comment:
It would be very good to add some explanation what is the expression syntax one can use here. Otherwise, there's no place to learn it from.

Are there plans to add it, if not now then soon?

Reply:
Yes, I plan to add docs for that. We also need to assess how long we do for docs as they currently don't work well in the panel.